ui/src/app/views/app/component.css

```diff
@@ -36,3 +36,19 @@
 .image-link-wrapper {
   text-decoration: none;
 }
+
+.top-nav {
+  border-bottom: 1px solid #ccc;
+  display: flex;
+}
+
+.btn.tab-link {
```

Comment:
Is the `btn` class necessary here? Maybe `tab-link` is specific-enough. I just mean semantically, not "would it work?"